ui/src/components/common/SlidingSidebar/partials/VertexDetails/partials/ProcessingRates/index.tsx

```diff
@@ -39,14 +39,14 @@ export function ProcessingRates({
       rates.push({
         partition: index,
         oneM: item.processingRates["1m"]
-          ? item.processingRates["1m"].toFixed(2)
```

Comment:
Instead of changing the rates and their types, can you add the "/sec" text to the JSX on line number : 85.
That way we can keep the data types as it is.